client.go

```diff
@@ -278,17 +278,11 @@ func Admin() Option {
 	}
 }
 
-// Options for specifying master resource name (e.g. "/hbase/master")
-func SetMaster(master zk.ResourceName) Option {
+// Option for specifying ZooKeeper path (e.g. "/hbase")
```

Comment:
Let's have the doc comments in the same style as other doc comments for options. They follow https://golang.org/doc/effective_go.html#commentary.